test/restart/mzcompose.py

```diff
@@ -63,6 +63,28 @@ def workflow_audit_log(c: Composition) -> None:
         raise Exception("audit logs not equal after restart")
 
 
+def workflow_timelines(c: Composition) -> None:
```

Comment:
@philip-stoev I tried to model this after the failing kafka test. Let me know if there's anything unnecessary/missing here. I tested this test on `main` and it fails.

Reply:
You need to also mention this test under `workflow_default` as otherwise it will not get run.

A mention of the ticket number in a comment or in the name of the workflow would also be a good idea.

Reply:
Fixed